Cargo.toml

```diff
@@ -35,6 +35,7 @@ cargo-registry-index = { path = "cargo-registry-index" }
 cargo-registry-markdown = { path = "cargo-registry-markdown" }
 cargo-registry-s3 = { path = "cargo-registry-s3" }
 chrono = { version = "=0.4.19", features = ["serde"] }
+cidr-utils = "=0.5.6"
```

Comment:
would it be possible to use https://github.com/achanda/ipnetwork instead? in that case it would be compatible with `diesel` (see https://github.com/diesel-rs/diesel/blob/master/diesel/Cargo.toml#L27) if we ever needed to store these in the database.

Reply:
Replaced [cidr-utils](https://crates.io/crates/cidr-utils) with the [ipnetwork](https://github.com/achanda/ipnetwork) crate.